src/app/services/BotService/bot.service.ts

```diff
@@ -0,0 +1,47 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+import { map } from 'rxjs/operators';
+import { environment } from '../../../environments/environment';
+import { CustomResponse } from './CustomResponse';
+
+@Injectable()
+export class BotService {
+  url = 'http://localhost:5005/conversations/default/parse';
```

Comment:
this won't work at all in production environment..  this should be configured in the environments files.